qemu/tests/cfg/fio_perf.cfg

```diff
@@ -1,5 +1,5 @@
 - fio_perf:
-    only x86_64,ppc64,ppc64le
+    only x86_64,ppc64,ppc64le,aarch64
```

Comment:
How about `no s390x`, because only one architecture does not supported.

Reply:
> How about `no s390x`, because only one architecture does not supported.

It's ok for me. Done.